invenio_rdm_records/services/schemas/parent/__init__.py

```diff
@@ -37,7 +37,7 @@ class RDMParentSchema(ParentSchema, FieldPermissionsMixin):
         "review": "review",
     }
 
-    access = fields.Nested(ParentAccessSchema, dump_only=True)
+    access = fields.Nested(ParentAccessSchema)
```

Comment:
Here I think now we opened the pandora box of people being able to update the whole access schema of the parent and definitely we don;t want that 😅  I think you should ensure that the `ParentAccessSchema` allows only the settings to be updated and all the other fields should remain `dump_only`

Reply:
You're right! I made all the fields of `ParentAccessSchema`  `dump_only`, except for the settings.